AdminWebsite/AdminWebsite/ClientApp/src/app/shared/footer/footer.component.html

```diff
@@ -1,7 +1,7 @@
 <footer class="govuk-footer">
   <div class="govuk-width-container">
-    <div class="govuk-footer__meta">
-      <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
+    <div class="govuk-footer__meta govuk-grid-row">
```

Comment:
You can achieve the same thing using `govuk-!-margin-bottom-0`

Reply:
done